pkg/wshrpc/wshremote/wshremote.go

```
@@ -230,68 +230,53 @@ func (impl *ServerImpl) RemoteStreamFileCommand(ctx context.Context, data wshrpc
 			return
 		}
-		defer tarWriter.Close()
-		logPrintfDev("creating tar stream for %q\n", path)
-		if finfo.IsDir() {
-			logPrintfDev("%q is a directory, recursive: %v\n", path, recursive)
-			if !recursive {
-				rtn <- wshutil.RespErr[[]byte](fmt.Errorf("cannot create tar stream for %q: %w", path, errors.New("directory copy requires recursive option")))
-				return
-			}
-		}
 		err := filepath.Walk(path, func(file string, fi os.FileInfo, err error) error {
```

golangci-lint (1.62.2)

SA4009: argument err is overwritten before first use

(staticcheck)